src/main/resources/interface-specification.yml

```
@@ -417,6 +417,53 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /tenants/{tenantId}/attributes/verified/updateRenewalStrategy:
```

this could be under the existing endpoint
```suggestion
  /tenants/{tenantId}/attributes/verified/{attributeId}:
```
as a matter of facts, it is an update of the entire verified attribute

(feel free to create a new payload if you think that having the attributeId both in the path and in the body is not a good idea)